README.md

```diff
@@ -1057,6 +1057,8 @@ Events published by the widget. Subscribe to these events using [on](#onevent-ca
 
 # Developing the Sign-In Widget
 
+We use Yarn as our node dependency manager. If you don't know what Yarn is, please check its [website](https://yarnpkg.com).
```

Comment:
Would probably skip the know part and just tell them how to install, i.e:

We use Yarn as our node package manager. To install Yarn, check out their [install documentation](https://yarnpkg.com/en/docs/install).